src/products/android-l10n/mobile_l10n_faqs.md

```diff
@@ -0,0 +1,139 @@
+# Firefox Mobile L10N FAQs
+
+This document includes a series of frequently asked questions about Firefox mobile localization across platforms. While it is primarily intended for developers, it also provides valuable context for product managers and engineering program managers.
+
+If you don't see your question addressed here, check also the main [Firefox L10n FAQ](../firefox_desktop/firefox_l10n_faqs.md) or look into our Developer best practices [documentation](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html). You’re also welcome to file an [issue](https://github.com/mozilla-l10n/documentation/issues) so we can add your question to this FAQ.
+
+## General
+
+### Who can I contact if I have more questions?
+
+A Program Manager (Localization ePM) is assigned to any Mozilla project that is localized through community. You can find this information in [Pontoon](https://pontoon.mozilla.org/projects/firefox/) for any project in the header section, and more information about the localization team is available in the Confluence page dedicated to [Localization](https://mozilla-hub.atlassian.net/wiki/spaces/FDPDT/pages/11045331/Localization+l10n+and+Translation) (available to Mozilla staff only).
+
+### [Should I enable my feature only for English, or ship it in English for everyone?](../firefox_desktop/firefox_l10n_faqs.md#should-i-enable-my-feature-only-for-english-or-ship-it-in-english-for-everyone)
```

Comment:
I wonder if this looks weird. Would it work better to have the title, then a list "See answer here"?

Reply:
I agree, I will update this as part of the rest